.taskmaster/scripts/normalize_task_prompts.py

```
@@ -0,0 +1,119 @@
+    "**Write-set:**",
+    "**Acceptance:**",
+    "**Verify:**",
+    "**Stop conditions / terminal states:**",
+]
+
+REQUIRED_FOR_VALIDATION = ["Load context", "Acceptance", "Verify"]
+
+
+def trim_kontext(details: str) -> str:
+    """Remove trailing whitespace from each line, collapse 3+ blank lines to 2."""
+    lines = [l.rstrip() for l in details.splitlines()]
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)